offchain/state-server/src/rollups_state_server.rs

```diff
@@ -1,15 +1,34 @@
-use anyhow::Result;
+// Copyright Cartesi Pte. Ltd.
+//
+// SPDX-License-Identifier: Apache-2.0
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
 
 use types::foldables::authority::rollups::RollupsState;
 
 #[tokio::main]
-async fn main() -> Result<()> {
+async fn main() {
```

Comment:
Let's standardize these main functions as well. For instance, the inspect-server main function returns a box dyn error and doesn't need these nested matches. It looks clearer.

https://github.com/cartesi/rollups/blob/0549b263a03883a8bbce6fbabe71f6bec6a52c91/offchain/inspect-server/src/main.rs#L16

If we use box Dyn, we don't need to print the error message with tracing; it already does that for us. However, we need to confirm that it prints the context as well.

Reply:
Ok, I'm going to run a few tests here to see the behavior of the two errors handling and come back to this topic with more info later.